src/Services/FHIR/FhirDocRefService.php

```diff
@@ -152,8 +152,14 @@ private function getMostRecentCCDReference($oeSearchParameters): FHIRDocumentRef
 
         // document create event
         $event = new PatientDocumentCreateCCDAEvent($patient['pid']);
-        $event->addSection("continuity_care_document"); // make it a CCD
 
+        // apparently the PHP CCDA just sends over every single section/component regardless of whether its used or not
+        // the node server on the backend is what distinguishes between what sections to include / exclude based on the
+        // document type that is sent.  Bizarre.
+        // TODO: if we ever decide to actually put the filtering on the PHP side we will need to adjust this.
```

Comment:
Not true. If sections are not sent to generator the generator will not render. What's so bizarre about that?
![image](https://user-images.githubusercontent.com/1859985/174489798-64e8e391-97a8-4b51-a97b-51dc59fbb494.png)